hw/drivers/pwm/pwm_nrf52/src/pwm_nrf52.c

```diff
@@ -545,8 +548,13 @@ nrf52_pwm_disable(struct pwm_dev *dev)
         return (-EINVAL);
     }
 
+    if (!instances[inst_id].playing) {
+        return (-EINVAL);
```

Comment:
EINVAL usage is inconsistent in this file some functions return -EINVAL some EINVAL. I don't know which should be used but having both version looks kind of strange.

Reply:
It is stated on the comments that this function returns a negative. However we can go with the non zero(i.e. use non-negative error codes) 
in this case becausee we got no other data being returned.